src/test/resources/for-execution/augmentations-with-fallback.golo

```diff
@@ -0,0 +1,34 @@
+module golotest.execution.AugmentationsWithFallback
+
+import java.util.stream.Collectors
+
+augment java.lang.String {
+  function fallback = |this, name, args...| -> "Fallback for this " + this + " named " + name + " with args" + args: asList()
+}
+
+augment Fluent {
```

Comment:
No it's not. It just a "bug" in the names of the generated classes that are the same in this precise case.